R/swagger.R

```diff
@@ -17,34 +17,32 @@ plumberToSwaggerType <- function(type){
 #' Convert the endpoints as they exist on the router to a list which can
 #' be converted into a swagger definition for these endpoints
 #' @noRd
-prepareSwaggerEndpoints <- function(routerEndpoints){
+prepareSwaggerEndpoints <- function(routerEndpointEntries){
   endpoints <- list()
 
-  for (fil in routerEndpoints){
-    for (e in fil){
-      # TODO: we are sensitive to trailing slashes. Should we be?
-      cleanedPath <- gsub("<([^:>]+)(:[^>]+)?>", "{\\1}", e$path)
-      if (is.null(endpoints[[cleanedPath]])){
-        endpoints[[cleanedPath]] <- list()
-      }
+  for (e in routerEndpointEntries){
+    # TODO: we are sensitive to trailing slashes. Should we be?
```

Comment:
I think we should be, but I can imagine it being an option.

Reply:
that is an old comment from https://github.com/trestletech/plumber/pull/280/files#diff-32592aa9d638876dfcfa9d16765e4311L25 

I think we should be as well